deploy/templates/base/kustomization.yaml

```diff
@@ -13,7 +13,7 @@ configMapGenerator:
 
 patchesStrategicMerge:
 - manager_image_patch.yaml
-- manager_dev_mode_patch.yaml
+# - manager_dev_mode_patch.yaml
```

Comment:
why did you comment it? does it make sense to disable dev mode by default and have the ability to enable it with makefile?

Reply:
I commented it out since it should be disabled by default. I'll remove the commented line and we can look into enabling it for local builds only. For `make run` and `make debug`, we should be able to set the env var in the terminal and have it be picked up, making this patch unnecessary.